Marlin/Marlin_main.cpp

```diff
@@ -1599,7 +1673,7 @@ static void setup_for_endstop_move() {
       // move back down slowly to find bed
       set_homing_bump_feedrate(Z_AXIS);
 
-      zPosition -= home_bump_mm(Z_AXIS) * 2;
+      zPosition = MIN_PROBE_PT;
```

Comment:
Yes, on re-bump this might work. But we don't really know that `zPosition` is accurate. If you only wanted to keep the probe from going more than 2mm below the bed, you could use…
```cpp
zPosition -= home_bump_mm(Z_AXIS) + 2;
```